### ISSUE TITLE
Ability to switch of page editor options

### ISSUE THREAD

**Issue by CraigMac1000** [beaverbuilder/feature-requests#104]
Hi!
I'm starting to use Gutenberg more often as  sometimes have an issue whereby the client clicks the "beaver builder" option by mistake or vice verse clicks 'Use standard editor' and this causes obvious issues. 

That being said I would still like to use BB modules and add them to pages using the short-code. I realize by un checking the  Builder Access option for the editor I am able to hide BB from this role but it would be useful if I could still allow access to saved rows column and modules so for example, if I was using an accordion the client could still go in and edit the saved modules, rows or whatever.  But they cant norse up the page! 

In other words could there be a way to completely disable BB as a page editor, so remove all page level options (potentially for all roles) and just have the Saved Rows, Columns and Modules accessible? 

I'm not sure that makes sense but I gave it my best shot! 

Thanks!

**Comment by CraigMac1000** [comment]
Duh! Figured out there is a way soon as I hit submit. 

I'll get my coat!

**Comment by CraigMac1000** [comment]
OK! I closed this as I thought I'd cracked it but turns out although I can show the admin menu to 'editor role', if I want to disable them from turning BB on at page level I have to disable 'Builder Access' which means they then can see the saved rows,modules etc, but they cant edit.
